Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift

```diff
@@ -0,0 +1,58 @@
+import Capture
+import CocoaLumberjack
+import os
+
+/// The wrapper around Capture SDK logger that conforms to `DDLogger` protocol from `CocoaLumberjack`
+/// library and can be used as a drop-in solution for forwarding `CocoaLumberjack` logs to bitdrift
+/// Capture SDK.
+final class CaptureDDLogger: NSObject, DDLogger {
+    private let osLogger = OSLogger()
+
+    var logFormatter: DDLogFormatter?
+
+    func log(message logMessage: DDLogMessage) {
+        guard let level = LogLevel(logMessage.level) else {
+            return
+        }
+
+        Capture.Logger.log(
+            level: level,
+            message: logMessage.message
```

Comment:
fyi on Android I decided to attach an extra filed `_source` with some name that identifies the library that is sending the logs ("Timber" in my case)

Reply:
how are they not internal? In my view "internal" means anything added automatically by code that we provide, which is the case here, wouldn't you agree?